src/typegraph/rust/typegraph/src/utils.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright Metatype OÜ, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+
+use serde::Serialize;
+
+pub fn json_stringify<S>(value: &S) -> String
+where
+    S: Serialize + ?Sized,
+{
+    serde_json::to_string(value).unwrap()
+}
```

Comment:
Since I'd expect this to be used a lot, maybe use`.context(format!("Failed {:?}", value))` or return a result?

Reply:
nvm, it has `Serialize` trait.

Reply:
But you're right we should add more context there.